axiom-eth/src/rlp/mod.rs

```diff
@@ -271,6 +333,108 @@ impl<'range, F: ScalarField> RlpChip<'range, F> {
         RlpArrayPrefixParsed { is_big, next_len, len_len }
     }
 
+
+    pub fn parse_rlp_prefix(
+        &self,
+        ctx: &mut Context<F>,
+        prefix: AssignedValue<F>,
+    ) -> RlpPrefixParsed<F>{
+        let is_field = self.range.is_less_than(
+            ctx,
+            Existing(prefix),
```

Comment:
We did a rust trick so specifically for `Existing` you don't need to type it and it will auto-infer to add `Existing()`